src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -102,13 +102,13 @@ public boolean canCheck( final Object toVerify ) {
 	}
 
 	private Set<RootElement> convert( final WebDriver driver, final RemoteWebElement webElement ) {
+		final BufferedImage screenshot = shoot( driver, webElement );
```

Comment:
Good idea, this should work. 👍 But maybe we should add a comment that the order matters? Or adapt the `shoot` method name that it possibly scrolls? Otherwise, one may change the order or inline the variable again?